app/controllers/Setup.scala

```diff
@@ -173,13 +173,23 @@ final class Setup(
               _ ?? { game =>
                 for {
                   blocking <- ctx.userId ?? env.relation.api.fetchBlocking
-                  hookConfig = lila.setup.HookConfig.default(ctx.isAuth) withRatingRange get(
-                    "rr"
-                  ) updateFrom game
+                  hookConfig = lila.setup.HookConfig.default(ctx.isAuth)
+                  hookConfigWithRating = get("rr").fold(
+                    hookConfig.withRatingRange(
+                      ctx.me.fold(0.some)(_.perfs.ratingOf(game.perfKey)),
```

Comment:
`0.some` means there is a rating of zero points? probably shouldn't

Reply:
Ya, this is very arbitrary, just to handle the anonymous case. I brought in the Glicko default rating at one point, maybe that's better.